docs/guide-forms.md

```diff
@@ -37,7 +37,7 @@ First we will start by adding the form to our template tag:
   </p>
   <div class="sender-info">
     <div>
-      <label for="name" class="label" >Your name</label>
+      <label for="name" class="label">Your name</label>
```

Comment:
Changes should conform the PR title. Single space it's not a big issue. I would suggest `eslint` should be the part of `gridsome`